server/server.js

```diff
@@ -293,5 +383,10 @@ app.use(function(err, req, res, next) {
 });
 
 // Start the application.
-var server = app.listen(3000);
-server.setTimeout(1200000);
+var privateKey = fs.readFileSync(privateKeyPath);
+var certificate = fs.readFileSync(certificatePath);
+var httpsServer = https.createServer({
+  key: privateKey,
+  cert: certificate
+}, app).listen(443);
```

Comment:
If you are listening to 443, then all the queries are to be sent to 443, aren't they?